.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
```

Comment:
let's move this file from root to .github

Reply:
Is this a strong preference? Having it in the root directory is the convention and any contributor thinking about pre-commit hooks would expect it there. I think moving it to .github would defeat the point as no one would adapt using it.

Reply:
sorry - miss that this is pre-commit config. 

But in general i would move towards clean and tidy first level directory layout that would feet on one screen vs inflating it with more files

Reply:
I agree. Refactoring the requirements management will help a lot here!

Reply:
yes. and in case on this repo - probably at some point doing a better split/separation of daal4py/scikit-learn-intelex